src/config.ts

```diff
@@ -40,7 +42,17 @@ export const configConstants = {
     stage: "dev",
     prefix: "sls",
     localPort: 7071,
+    os: FunctionAppOS.WINDOWS,
   },
+  dockerImages: {
```

Comment:
why are these config under `dockerImages`?

Reply:
That's where we've been keeping any configuration-related constants. Linux function apps are built off of docker images that correspond to the configured runtime